vampyr/CMakeLists.txt

```diff
@@ -1,18 +1,33 @@
 # create python module
-pybind11_add_module(vampyr3d MODULE THIN_LTO "")
+pybind11_add_module(vampyr3d MODULE NO_EXTRAS "")
```

Comment:
What did the `NO_EXTRAS` do? Do we still need it? @robertodr

Reply:
It fully deactivated link-time optimization. We can put `THIN_LTO` back, I think. In Psi4 LTO broke a lot of things, so we should watch out for weird behavior.